Editorial review for chapters 6 and 7 of the User Interface Guide (#870)
windup/windup-documentation: docs/topics/mta-web-tagging-an-application.adoc

[id="mta-web-tagging-an-application_{context}"]
= Tagging an application

You can attach various tags to the application that you want to analyze. You can use tags to classify applications and instantly identify application information, for example, an application type, data center location, and technologies used within the application. You can also use tagging to associate archetypes to applications for automatic assessment. For more information about archetypes, see xref:mta-archetypes_user-interface-guide[Working with archetypes].

> Tagging can be either xref:manual-tagging-of-an-application_user-interface-guide[manual] or xref:automatic-tagging-of-an-application_user-interface-guide[automatic].

Reviewer: ```suggestion
Tagging can be done xref:automatic-tagging-of-an-application_user-interface-guide[automatically] during the analysis xref:manual-tagging-of-an-application_user-interface-guide[manually] at any time.
```

> NOTE: Manual and automatic tagging are not interchangeable but complementary. For example, an analysis can only tag the application based on its technologies. If you want to tag the application also with the location of the data center where it is deployed, you need to tag the application manually.

Reviewer: I don't really understand the nuance here in the note, which seems pretty important. What do you mean by "complimentary"? In the example, is the analysis tagging technologies automatic or manual? Why is tagging by location only possible manually?

Reviewer: Ok. Couple of things:
1. The text makes it sound like tagging, both auto and manual, happens before the analysis, not during. Recommend rewriting the first sentence of the section to clarify timing.
2. Recommend clarifying which tags can be assigned automatically and which can't.

Author: @abrugaro, could you please check Miriam's comment above?

Reviewer 2: Hi, sure @mpershina , @mportman12

1. Auto tagging happens during the analysis, manual tagging can happen anytime (before, during or after the analysis)

2. That question is a bit more difficult to answer, I'll investigate and come back with an answer

Author: Thanks! As for the first point, Miriam's suggestion works :-) Let's see about the second one.

> NOTE: Manual and automatic tagging are not interchangeable but complementary. For example, an analysis can only tag the application based on its technologies. If you want to tag the application also with the location of the data center where it is deployed, you need to tag the application manually.

Reviewer: ```suggestion
NOTE: Not all tags can be assigned automatically. For example, an analysis can only tag the application based on its technologies. If you want to tag the application also with the location of the data center where it is deployed, you need to tag the application manually.
```
